### ISSUE TITLE
Numbering grammatical homonyms

### ISSUE THREAD

**Issue by anacastrosalgado** [DARIAH-ERIC/lexicalresources#50]
In Portuguese Academy Dictionary, I have grammatical homonyms and I have to number the entry.

What's the best way to encode the number? Using `<lbl>` or `<num>`?

capital:1 adj. m. e f.
capital:2 n. f.

```
     <entry xml:id="DACL.CAPITAL:1" xml:lang="pt">
               <form type="lemma">
                  <orth>capital</orth>
                  <lbl>:1</lbl>
```

 ```
   <entry xml:id="DACL.CAPITAL:2" xml:lang="pt">
               <form type="lemma">
                  <orth>capital</orth>
                  <lbl>:2</lbl>

```

**Comment by xlhrld** [comment]
As this really serves as a label that just happens to come in the form of a number (could be a letter or some other character as well) I'd opt for `lbl` like you propose. I'd separate the punctuation from it with `pc`, though. I'd also try to keep the whitespace in sync with the print but YMMV:
```
<form type="lemma">
  <orth>capital</orth><pc>:</pc>
  <lbl>1</lbl>
  <gramGrp>
    <gram type="partOfSpeech">adj.</gram>
    <gram type="gender">m. e f.</gram>
  </gramGrp>
</form>
```

Using `form/@n="1"` for the homonym marker may also be an option if you do not necessarily have or want to keep it as a text node.

**Comment by anacastrosalgado** [comment]
@xlhrld I agree with you. It could be a letter, a symbol or some other character. @ttasovac, do you agree on using lbl?

I'm not using <pc>, but I'll change it. Thanks.

**Comment by anacastrosalgado** [comment]
Like this? If the purpose is to simplify, I think I'm complicating.

**capital** (homonymicEntry and it's a simple word)

`<entry type =homonymicEntry" xml:id="DACL.CAPITAL:1" xml:lang="pt">`
`<form` `type="lemma">`
`<orth` `norm="capital">`
`<w` `type="simple">capital</w>`
`<orth>capital</orth><pc>:</pc>`
`<lbl>1</lbl>`
`<pron>kɐpitˈał</pron>`
`<gram type="partOfSpeech">adj.</gram>`
`<gram type="gender">m. e f.</gram>`
`</gramGrp>`
`</form>`

**Comment by TomazErjavec** [comment]
Hi,
my 2c:
* for machine processing, it would indeed be nice if `@n` were used to give the distinctive label for each homonym (maybe something to add to the specs?)
* but I think `@n` belongs on the `entry` element, not `form`, as it is the property of the whole entry
* if you want also to keep the form/content of the original dictionary, then `pc+lbl` makes sense
* but it would be nice if `lbl` were typed, so it is clear what kind of label this is (again, maybe something for the specs)
* also, `pc` could include spaces were required, so the result could be simply type-set
* not sure what you are trying to do with `w`, in fact, the previous example is not well-formed.

In short, I'd propose:
 
```
<entry type="homonymicEntry" n="1" xml:id="DACL.CAPITAL:1" xml:lang="pt"> 
  <form type="lemma">
       <orth><w type="simple">capital</w></orth>
       <pc>: </pc> <lbl type="homonymNumber>1</lbl> 
        ...
```

**Comment by bansp** [comment]
Minor remark: the ID must be "non-colonized", so the ":" should cause an error. Use a hyphen instead.
And as Tomaž says, there's a bit of ill-formedness above, one closing, and one opening tag missing.

**Comment by anacastrosalgado** [comment]
In Portuguese Academy Dictionary, it is encoded as the attribute n (number) in the entry element:

`         <entry type="derivativeWord" xml:lang="pt" xml:id="antepassado.1" n="1">`
